build/crds/reliablesyncs/cluster_objectsync_v1alpha1.yaml

```diff
@@ -10,6 +10,8 @@ spec:
     kind: ClusterObjectSync
     plural: clusterobjectsyncs
   scope: Cluster
+  subresources:
```

Comment:
why add `subresources ` ？

Reply:
Because we put the resourceversion in status, the updatestatus method needs the `status` subresource.  Xref: https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#subresources